packages/nouns-webapp/src/components/NavBarTreasury/index.tsx

```diff
@@ -0,0 +1,45 @@
+import classes from './NavBarTreasury.module.css';
+import { NavBarButtonStyle } from '../NavBarButton';
+
+interface NavBarTreasuryProps {
+  treasuryBalance: string;
+  treasuryStyle: NavBarButtonStyle;
+}
+
+const NavBarTreasury: React.FC<NavBarTreasuryProps> = props => {
+  const { treasuryBalance, treasuryStyle } = props;
+
+  let treasuryStyleClass;
+  switch (treasuryStyle) {
```

Comment:
Smell here: effectively duplicate code translating a `NavBarButtonStyle` to classes with `packages/nouns-webapp/src/components/NavBarButton/index.tsx`. Should be addressable with a single helper and a common module.

Doesn't need to block promotion but should be addressed later.